src/routes/analytics/new/NewDashboardPage.tsx

```diff
@@ -0,0 +1,39 @@
+import { useAtom } from "jotai"
+import { Flex } from "@mantine/core"
+import {
+  CreateDashboardModal,
+  EditableDashboard,
+} from "@metabase/embedding-sdk-react"
+import { navigate } from "wouter/use-browser-location"
+
+import { RemountOnSiteChange } from "../../../components/RemountOnSiteChange"
+
+import { siteAtom } from "../../../store/site"
+import { createDashboardIdAtom } from "../../../store/create"
+
+import { SANDBOXED_USER_GENERATED_COLLECTIONS } from "../../../constants/collections"
+
+export const NewDashboardPage = () => {
+  const [site] = useAtom(siteAtom)
+  const [dashboardId, setDashboardId] = useAtom(createDashboardIdAtom)
+
+  const collectionId = SANDBOXED_USER_GENERATED_COLLECTIONS[site]
```

Comment:
Ensures that the dashboard is saved in the "User-Generated" folder under the site's sandboxed collections by default. If the user tries to change the collection, it wouldn't work as they cannot save as per the permissions configuration.

Reply:
just a note that we don't seem to do this for when creating a question.

Somehow `Our analytics` is set by default if we don't click on the collection picker, and it will fail to save. If this needs the `CreateComponent` component to change then ignore it for this PR.

![Screenshot 2024-09-06 at 4 39 54 PM](https://github.com/user-attachments/assets/1fb00e4c-43cf-4671-b5c3-fc95a81ef66d)

Reply:
@WiNloSt Good point, I think `CreateQuestion` component doesn't have this prop as you mentioned. cc @oisincoveney do you think it make sense to have the `initialCollectionId` prop in `CreateQuestion` also, ot be consistent with `CreateDashboardModal`?